kaggle/models/model_new_instance_request.py

```diff
@@ -159,7 +159,7 @@ def framework(self, framework):
         """
         if framework is None:
             raise ValueError("Invalid value for `framework`, must not be `None`")  # noqa: E501
-        allowed_values = ["tensorFlow1", "tensorFlow2", "tfLite", "tfJs", "pyTorch", "jax", "flax", "pax", "maxText", "gemmaCpp", "tensorRtLlm", "ggml", "gguf", "coral", "scikitLearn", "mxnet", "onnx", "keras", "transformers", "other"]  # noqa: E501
+        allowed_values = ["tensorFlow1", "tensorFlow2", "tfLite", "tfJs", "pyTorch", "jax", "flax", "pax", "maxText", "gemmaCpp", "tensorRtLlm", "ggml", "gguf", "coral", "scikitLearn", "mxnet", "onnx", "keras", "transformers", "triton", "other"]  # noqa: E501

```

Comment:
The files under `kaggle/` are generated (build output). I am discussing with Steve about removing them from the repo to avoid this confusion.

Some file are copied from `/src/kaggle` at build time. Others (like this one), are generated by Swagger from the KaggleSwagger.yaml configuration file.

tl;dr this change will be overridden at the next release. Updating `KaggleSwagger.yaml` was enough.

cc/ @dolaameng @stevemessick



For instance, this file is generated from the